plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerAnimator.java

```diff
@@ -304,6 +343,21 @@ private void playCompassAnimators(long duration) {
     compassAnimatorSet.start();
   }
 
+  private void updateAccuracyAnimators(float targetAccuracyRadius, float previousAccuracyRadius) {
+    cancelAccuracyRadiusAnimations();
+    accuracyRadiusAnimator = new AccuracyAnimator(previousAccuracyRadius, targetAccuracyRadius);
+    accuracyRadiusAnimator.addUpdateListener(accuracyRadiusUpdateListener);
+  }
+
+  private void playAccuracyRadiusAnimators(long duration) {
+    List<Animator> accuracyAnimators = new ArrayList<>();
+    accuracyAnimators.add(accuracyRadiusAnimator);
+    AnimatorSet accuracyAnimatorSet = new AnimatorSet();
```

Comment:
@LukasPaczos do we need to create an `AnimatorSet` here if the `accuracyRadiusAnimator` is the only one being added?